### ISSUE TITLE
Hardness Setting in Brush and Clone Stamp Tools

### ISSUE THREAD

**Issue by khatfull** [PintaProject/Pinta#751]
**Description**
This is currently the one thing I miss from Windows-based paint.net, the hardness slider in the clone stamp and brush tools.  I used these settings a LOT during retouching.

Any chance of adding the paint.net style  hardness setting to these tools?  It's much akin to the tolerance setting in other tools, not too bad to implement hopefully?

Thanks.

**Comment by cameronwhite** [comment]
Yeah this would be nice to add. It needs some investigation to see if the Cairo graphics API has a nice way to do this

**Comment by ryan-de-boer** [comment]
+1 for this
I implemented smooth brushes (and smooth stamp) and discuss them here:
https://github.com/PintaProject/Pinta/discussions/1984

It should be easy to be able to tweak the smoothness (hardness) with a slider.
I want to tweak them too!

**Comment by PlatinumSnow** [comment]
+1 for hardness slider, especially for the brush tool.
